File: 2186-minimum-number-of-steps-to-make-two-strings-anagram-ii/2186-minimum-number-of-steps-to-make-two-strings-anagram-ii.java
class Solution {
    public int minSteps(String s, String t) {
        int[] count = new int[26];
        int min = 0,i;
        for(i=0;i<s.length();i++)count[s.charAt(i)-'a']++;
        for(i=0;i<t.length();i++)count[t.charAt(i)-'a']--;
        for(i=0;i<26;i++)min += Math.abs(count[i]);
        return min;
    }
}

File: 2186-minimum-number-of-steps-to-make-two-strings-anagram-ii/2186-minimum-number-of-steps-to-make-two-strings-anagram-ii.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        int count[26]={0},min=0;
        for(int i=0;i<s.length();i++){
            count[s[i]-'a']++;
        }
        for(int i=0;i<t.length();i++){
            count[t[i]-'a']--;
        }
        for(int i=0;i<26;i++){
            min+=abs(count[i]);
        }
        return min;
    }
};